src/v/cluster/scheduling/allocation_node.h

```diff
@@ -30,6 +30,7 @@ class allocation_node {
     enum class state { active, decommissioned, deleted };
     using allocation_capacity
       = named_type<uint32_t, struct allocation_node_slot_tag>;
+    enum class deallocation_error_policy { strict, relaxed };
```

Comment:
Please include some comments about what this is about and reference the current issue.